eclipse/che
# make it possible to disable labeling the namespaces

Signed-off-by: Michal Vala <mvala@redhat.com>



### What does this PR do?
Adding che property to disable che-server to try to label the workspace namespaces. If it is enabled, we do not fail, but log warn message.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/18301

### Documentation PR
https://github.com/eclipse/che-docs/pull/1694


### How to test this PR?

1. deploy with `che.infra.kubernetes.namespace.label=true`
1. you should see either warning that che-server can't label the namespace, or if che SA has the permissions, the namespace should be labeled
1. set the `che.infra.kubernetes.namespace.label=false`
1. you should no longer see the warning, and new namespaces should not be labeled

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
